scripts/create_cloud_secrets.sh

```diff
@@ -60,24 +60,25 @@ subjectAltName = @alt_names
 [alt_names]
 DNS.1   = *.${namespace}
 DNS.2   = *.${namespace}.svc.cluster.local
-DNS.3   = *.pl-nats
-DNS.4   = pl-nats
-DNS.5   = *.local
-DNS.6   = localhost
-DNS.7   = kratos
+DNS.3   = *.pl-nats.${namespace}.svc
+DNS.4   = *.pl-nats
+DNS.5   = pl-nats
+DNS.6   = *.local
+DNS.7   = localhost
+DNS.8   = kratos
 EOS
 
 openssl genrsa -out ca.key 4096
 openssl req -new -x509 -sha256 -days 365 -key ca.key -out ca.crt -subj "/O=Pixie/CN=pixie.local"
 
 openssl genrsa -out server.key 4096
-openssl req -new -sha256 -key server.key -out server.csr -config ssl.conf -subj "/O=Pixie/CN=pixie.local"
+openssl req -new -sha256 -key server.key -out server.csr -config ssl.conf -subj "/O=Pixie/CN=pixie.server"
```

Comment:
Are we going to close that other PR that makes this change?

Reply:
Yeah it wasn't GPG signed and the author didn't respond.